### DISCUSSION TITLE
hsrb_moveit error

### DISCUSSION THREAD

**Discussion by datemitumasa** [HumanSupportRobot/QandA#37] [Q&A]
We used moveit, which is available in the community https://git.hsr.io/tmc/hsrb_moveit/-/tree/v0.6.3-community. When "Move group" is moved with "whole_body", an IK error occurs if the base_footprint is more than 1m away from the odom. 
[ERROR] [1671673962.355695997]: whole_body/whole_body: Unable to sample any valid states for goal tree
It seems that the reason is that the limit of "world_joint" in "virtual_joint" is set to 1.0. Does anyone know how to solve this problem?

**Comment by hsr-admin** [comment]
Try increasing the size of the workspace as follows.

![image](https://user-images.githubusercontent.com/103644635/211984865-2a08ae8e-d75c-42b3-9559-16a2c856d908.png)
